yarn.lock

```diff
@@ -16414,6 +16208,13 @@ fast-url-parser@1.1.3:
   dependencies:
     punycode "^1.3.2"
 
+fast-xml-parser@4.2.4:
```

Comment:
Why was this brought in? It has a security vulnerability. Dbl check other deps too, the vue build test is failing as well.

Reply:
This is fixed with the latest amplify-js tagged version.